peakinvestigator/internal/uploader.py

```diff
@@ -0,0 +1,80 @@
+## -*- coding: utf-8 -*-
+#
+# Copyright (c) 2016, Veritomyx, Inc.
+#
+# This file is part of the Python SDK for PeakInvestigator
+# (http://veritomyx.com) and is distributed under the terms
+# of the BSD 3-Clause license.
+from peakinvestigator.internal import *
```

Comment:
These imports shouldn't be necessary because you're assuming all of the methods of a `progress_factory` interface have been implemented.